test/threads_test.py

```diff
@@ -91,8 +108,9 @@ def todo_test_benchmark_workers(self):
         # You can pass in benchmark data, and functions if you want.
         # a_bench_func - f(data)
         # the_data - data to work on.
-
-        self.fail()
+        optimal_workers = threads.benchmark_workers()
+        self.assertTrue(type(optimal_workers) == int)
+        self.assertTrue(0 <= optimal_workers < 64)
```

Comment:
Not that worried about the test for a performance testing function but we could probably give the input parameters a fairly quick test here by adding another test underneath that looks a bit like:

```
def smooth_scale_bench(data):
    pygame.transform.smoothscale(data, (128, 128))

surf_data = [pygame.Surface((32, 32), 0, 32),
             pygame.Surface((64, 64), 0, 32)]
best_number = threads.benchmark_workers(smooth_scale_bench, surf_data)
self.assertTrue(isinstance(best_number,int))

Reply:
Good suggestion! I added a variant of this and it seems to work well.